x-pack/legacy/plugins/apm/public/components/app/Settings/CustomizeUI/CustomActions/Flyout/SettingsSection.tsx

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { EuiFieldText, EuiFormRow, EuiSpacer, EuiTitle } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+
+interface Props {
+  label: string;
+  setLabel: (label: string) => void;
```

Comment:
ideally, event handlers are named appropriately, like `onLabelChange`, that makes them more easily recognisable + makes it clear that the consumer of this component should decide how to handle the event.